keystorage/aws/src/test/java/tech/pegasys/signers/aws/AwsSecretsManagerTest.java

```diff
@@ -22,45 +22,47 @@
 import java.util.List;
 import java.util.Optional;
 import java.util.UUID;
-import java.util.stream.Collectors;
 
 import org.junit.jupiter.api.AfterAll;
 import org.junit.jupiter.api.AfterEach;
 import org.junit.jupiter.api.Assumptions;
 import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
 import org.junit.jupiter.api.Test;
-import org.junit.jupiter.api.TestInstance;
 import software.amazon.awssdk.auth.credentials.AwsBasicCredentials;
 import software.amazon.awssdk.auth.credentials.StaticCredentialsProvider;
 import software.amazon.awssdk.regions.Region;
 import software.amazon.awssdk.services.secretsmanager.SecretsManagerClient;
 import software.amazon.awssdk.services.secretsmanager.model.CreateSecretRequest;
+import software.amazon.awssdk.services.secretsmanager.model.CreateSecretResponse;
 import software.amazon.awssdk.services.secretsmanager.model.DeleteSecretRequest;
 import software.amazon.awssdk.services.secretsmanager.model.Tag;
 
-@TestInstance(TestInstance.Lifecycle.PER_CLASS)
 class AwsSecretsManagerTest {
 
-  private final String RW_AWS_ACCESS_KEY_ID = System.getenv("RW_AWS_ACCESS_KEY_ID");
-  private final String RW_AWS_SECRET_ACCESS_KEY = System.getenv("RW_AWS_SECRET_ACCESS_KEY");
-
-  private final String RO_AWS_ACCESS_KEY_ID = System.getenv("RO_AWS_ACCESS_KEY_ID");
-  private final String RO_AWS_SECRET_ACCESS_KEY = System.getenv("RO_AWS_SECRET_ACCESS_KEY");
-  private final String AWS_REGION = "us-east-2";
-
-  private AwsSecretsManager awsSecretsManagerDefault;
-  private AwsSecretsManager awsSecretsManagerExplicit;
-  private AwsSecretsManager awsSecretsManagerInvalidCredentials;
-  private SecretsManagerClient secretsManagerClient;
-  private String secretName;
-  private String secretNamePrefix;
-  private List<String> secretNames;
-  private AbstractMap<String, String> secretTags;
-
-  private static final String SECRET_VALUE =
+  private static final String RW_AWS_ACCESS_KEY_ID = System.getenv("RW_AWS_ACCESS_KEY_ID");
+  private static final String RW_AWS_SECRET_ACCESS_KEY = System.getenv("RW_AWS_SECRET_ACCESS_KEY");
+
+  private static final String RO_AWS_ACCESS_KEY_ID = System.getenv("RO_AWS_ACCESS_KEY_ID");
+  private static final String RO_AWS_SECRET_ACCESS_KEY = System.getenv("RO_AWS_SECRET_ACCESS_KEY");
+  private static final String AWS_REGION = "us-east-2";
+
+  private static AwsSecretsManager awsSecretsManagerDefault;
+  private static AwsSecretsManager awsSecretsManagerExplicit;
+  private static AwsSecretsManager awsSecretsManagerInvalidCredentials;
+  private static AwsBasicCredentials awsBasicCredentials;
+  private static StaticCredentialsProvider credentialsProvider;
+  private static SecretsManagerClient testSecretsManagerClient;
+  private static String testSecretName;
+  private static String testSecretNamePrefix;
+  private static List<String> testSecretNames;
+  private static AbstractMap<String, String> testSecretTags;
+
+  private final String SECRET_VALUE =
```

Comment:
nit: we don't care about the value so long as it returns the right one so could just make this a lot simpler with a smaller test value